dev/releases/release_notes.py

```diff
@@ -151,7 +157,8 @@ def get_pr_list(date: str, extra: str) -> List[Dict[str, Any]]:
         capture_output=True,
         text=True,
     )
-    return json.loads(res.stdout.strip())
+    jsonList = json.loads(res.stdout.strip())[::-1] # reverse the list
```

Comment:
Why do we reverse the list? Is that meant to ensure a certain sort order? Probably "PRs in ascending order of ID" as Lars suggested?

 But does `gh pr list` guarantee *anything* about the order? It seems to me it would be better to explicitly sort by whatever criterion we want.

Reply:
Another reason for having an explicit `sort` is code clarity. It is much easier to understand what's going on, and also is then obvious how to further tweak the sorting.